ldcbench.api/src/main/java/org/dice_research/ldcbench/graph/Graph.java

```diff
@@ -126,4 +126,11 @@ public default int incomingEdgeCount(int nodeId) {
      * @return the external ID of the node in the other graph.
      */
     public int getExternalNodeId(int nodeId);
+
+    /**
+     * Returns the ID of the first Blank Node of the graph
+     *
+     * @return the ID of the first Blank Node
+     */
+    public int getBlankNodesIndex();
```

Comment:
Overall, the idea of using the ID to decide whether it is a blank node or not is good. However, maybe the interface of the Graph class would be much easier to use, if you add another method as follows:
```java
public default boolean isBlankNode(int nodeId) {
    return nodeId >= getBlankNodesIndex();
}
```
This would ease the usage of the Graph class because the user of the class does not have to know how the internal IDs work. They only want to know whether a node is a blank node or not :wink: